codejail/proxy.py

```diff
@@ -113,7 +113,7 @@ def get_proxy():
     # If we need a proxy, make a proxy.
     if PROXY_PROCESS is None:
         # Start the proxy by invoking proxy_main.py in our root directory.
-        root = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        root = os.path.dirname(os.path.realpath(__file__))
```

Comment:
How this is a breaking change we aren't using PROXY process in codippy default sandbox 🤔

Reply:
I haven't actually looked at the codippy default sandbox at all. I didn't know `proxy_main.py` was only used in the codippy validation sandbox.

Do you think this PR can be merged now?

Reply:
Oh I spotted it, it's declared in the Docker container as ENV variable.